portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,10 +40,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Would you mind adding a comment in a Javadoc style to explain what this function is doing? All public functions should have documentation.

http://www.drjava.org/docs/user/ch10.html

Reply:
Solved